### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@
       -  **By:** Checking that node +h+ is available in Rancher
```
Comment (xansec) [inline tests/README.md]: This looks like a mistake? Could you please revert?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: tests/README.md (hunk 1 above)
Comment (bmwtsn098) [reply tests/README.md]: I think I've done that correctly. Let me know if anything else needs to be done

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**